pkg/controller/util/cluster_util.go

```diff
@@ -45,7 +45,7 @@ const (
 	DefaultLeaderElectionLeaseDuration = 15 * time.Second
 	DefaultLeaderElectionRenewDeadline = 10 * time.Second
 	DefaultLeaderElectionRetryPeriod   = 5 * time.Second
-	DefaultLeaderElectionResourceLock  = "configmaps"
+	DefaultLeaderElectionResourceLock  = fedv1a1.ConfigMapResourceLock
```

Comment:
(No action required) I'm not sure why it ever made sense for these defaults to live here.  Something to revisit when the admission webhook takes responsibility for defaulting.